source/time_tables/tests/departure_boards_test.cpp

```diff
@@ -559,12 +585,12 @@ BOOST_FIXTURE_TEST_CASE(test_calendar_with_impact, calendar_fixture) {
 
     using btp = boost::posix_time::time_period;
 
-    navitia::apply_disruption(b.impact(nt::RTLevel::Adapted, "Disruption stop1")
+    navitia::apply_disruption(b.impact(nt::RTLevel::Adapted, "Disruption stop2")
```

Comment:
The best here (now that you added fork route) would be to disrupt on `StopR1`, and check below that on `StopR2` you have the 2 vj at `10:30`, when using calendar `not_associated_cal`.
To discuss if it's a pain.